src/main/java/com/promise8/wwwbe/model/dto/MeetingMainGetResDtoWrapper.java

```diff
@@ -0,0 +1,127 @@
+package com.promise8.wwwbe.model.dto;
+
+import com.promise8.wwwbe.model.entity.MeetingEntity;
+import com.promise8.wwwbe.model.entity.MeetingPlaceEntity;
+import com.promise8.wwwbe.model.entity.MeetingUserEntity;
+import com.promise8.wwwbe.model.entity.MeetingUserTimetableEntity;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Setter
+@Getter
+@Builder
+public class MeetingMainGetResDtoWrapper {
+    private static class HostAndVotingCnt {
+        private Integer votingUserCount = 0;
+        private String hostName = null;
+    }
+
+    private static class ConfirmedPromise {
+        private LocalDate promiseDate = null;
+        private PromiseTime promiseTime = null;
+        private String promisePlace = null;
+        private HostAndVotingCnt hostAndVotingCnt = new HostAndVotingCnt();
+    }
+
+    private List<MeetingMainGetResDto> meetingMainIngGetResDtoList;
+    private List<MeetingMainGetResDto> meetingMainEndGetResDtoList;
+
+    public static MeetingMainGetResDtoWrapper of(List<MeetingEntity> meetingEntityList) {
+        List<MeetingMainGetResDto> meetingMainIngGetResDtoList = new ArrayList<>();
+        List<MeetingMainGetResDto> meetingMainEndGetResDtoList = new ArrayList<>();
+        for (MeetingEntity meeting : meetingEntityList) {
+            ConfirmedPromise confirmedPromise = new ConfirmedPromise();
+            if (MeetingStatus.DONE.name().equals(meeting.getMeetingStatus()) || MeetingStatus.CONFIRMED.name().equals(meeting.getMeetingStatus())) {
+                confirmedPromise = getConfirmedPromise(meeting.getMeetingUserEntityList(), meeting.getUserEntity().getUserId());
+
+                if (MeetingStatus.DONE.name().equals(meeting.getMeetingStatus())) {
+                    meetingMainEndGetResDtoList.add(MeetingMainGetResDto.of(
+                            meeting,
+                            meeting.getMeetingUserEntityList(),
+                            confirmedPromise.hostAndVotingCnt.votingUserCount,
+                            confirmedPromise.promiseDate,
+                            confirmedPromise.promiseTime,
+                            confirmedPromise.promisePlace,
+                            confirmedPromise.hostAndVotingCnt.hostName
```

Comment:
```suggestion
                    meetingMainEndGetResDtoList.add(MeetingMainGetResDto.of(meeting, confirmedPromise))
```
객체 참조가 대부분이라 위에처럼 줄여서 만드는게 좋을 것 같아용
+